events/class_models/possible_participant_model.py

```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from django.db.models import Q
 from django.core.exceptions import ValidationError
 from .meeting_models import OptionalMeetingDates
 from .participant_model import EventParticipant
```

Comment:
Can you merge this lines into one, like this 
```suggestion
from events.models import OptionalMeetingDates, EventParticipant
```

Reply:
Done, thanks 😄